README.md

```diff
@@ -1,5 +1,18 @@
 # TubiPlayer
-A media streaming video player based on ExoPlayer from google, with convenience methods for ad supported content
+The official android player of [Tubi TV](https://www.tubi.tv) -- A free movie and TV streaming service.
 
+# Purposes of TubiPlayer
+In the industry of streaming free multi-media contents to users, companies choose to show In-stream video ad to user between the main content as their business model. Examples: **YouTube**, **Tubi TV**.
 
-![alt text](https://github.com/Tubitv/TubiPlayer/blob/master/lib/doc/Screen%20Shot%202017-09-18%20at%204.23.53%20PM.png)
+Now from techonlogy side, to implement an android player that streams multiple media very well, also to switch between movies and ads dynamically, and finally to present with the seamless user experience. It is a series of stateful actions that poses a lot of complexities for enginners to manage. ***TubiPlayer offers industry standard multi-media streaming with Ad support solution on Android platform with highly customized code architecture and implementation***
```

Comment:
From the technology side of things, in order to implement an android player that streams multiple media very well, switches between movies and ads dynamically, and presents users with a seamless user experience, it requires a series of stateful actions that poses a lot of complexities for enginners to manage. ***TubiPlayer offers an industry standard, multi-media streaming and ad-supported solution on the Android platform, with highly customized code architecture and implementation***